app/assets/stylesheets/application.scss

```diff
@@ -637,3 +613,20 @@ input:focus ~ .highlight {
 .false {
   color: rgb(255,64,129);
 }
+
+.top {
+  padding: 16px;
+}
+
+.mdl-layout__header-row .mdl-navigation__link {
```

Comment:
Selector `mdl-layout__header-row` should be written in lowercase with hyphens<br>Selector `mdl-navigation__link` should be written in lowercase with hyphens